workflows_core/api/api.py

```diff
@@ -865,3 +874,35 @@ def _label_openai(
             params=params,
         )
         return get_response(response)
+
+    @retry()
+    def _aggregate(
```

Comment:
this should match the API and then abstracted away later.

Reply:
nws